apps/cms.c

```diff
@@ -1103,7 +1122,11 @@ int cms_main(int argc, char **argv)
             key = NULL;
         }
         /* If not streaming or resigning finalize structure */
-        if ((operation == SMIME_SIGN) && !(flags & CMS_STREAM)) {
+        if (operation == SMIME_SIGN && digestbin != NULL && !(flags & CMS_STREAM)) {
```

Comment:
This line is too long, can you please split it? Also better use `(flags & CMS_STREAM) == 0`.

Reply:
> This line is too long, can you please split it? Also better use `(flags & CMS_STREAM) == 0`.

To check for such issues in advance, one can use
```
util/check-format.pl apps/cms.c
```
(well, which on this file mourns about other, already existing style nits).